### PR TITLE
 Add address, telephone(s), fax(s), email, website

### PR DESCRIPTION
**Pull request by kenyiu**
Some more details are added to the output .json file, including:
- Address
- Telephone Numbers (if >1 no, then store in the 2nd column)
- Fax Numbers (If >1 no, then store in the 2nd column)
- Email
- Website

Cheers.

### CODE REVIEW

**Comment by siuying** [comment]
looks good, i made some comments on the commit, apart from that:
1. Can you add a small spec to test your changes, such that I can make sure i did not break it later
2. may be the telephone_1/telephone_2 and fax_1/fax_2 can become telephones and faxes which are array?

**Comment by kenyiu** [comment]
Thanks for comments.

For (1), may I know what do you mean by "small spec to test your changes"?

**Comment by siuying** [comment]
In https://github.com/OpenDataHK/legco_members/blob/master/spec/legco_spec.rb i made test to ensure each member has name, image, url and region. Perhaps add add address, telephone(s), fax(s), email and website?

**Comment by kenyiu** [comment]
OK, I can work on them. Thanks for comments. They did help!

**Comment by siuying** [comment]
Thanks! ;)

**Comment by kenyiu** [comment]
I've updated the code based on your recommendations. 
Check it out :)

File: lib/legco.rb [hunk 1]
```diff
@@ -24,23 +24,58 @@ def members
...
+        page = Nokogiri::HTML(open(url))
+        content = page.search("#_content_")
+        content.search("//table//table//tr").each do |tr|
+          case tr.search("td[1]").text
+          when "辦 事 處 地 址"
+            address = tr.search("td[3]").text.strip
+          when "辦 事 處 電 話"
+            temp = tr.search("td[3]").text.strip.delete(' ')
+            telephone[0] = temp.partition("/")[0].strip
+            telephone[1] = (temp.partition("/")[2].strip==""? nil : temp.partition("/")[2].strip)
```
Comment (siuying) [inline lib/legco.rb]: Prefer no nil in the telephone / tax...

``` ruby
telephone = []

# ...
telephone = tr.search("td[3]").split("/").collect {|phone| phone.delete(' ') }
```

Comment (kenyiu) [inline lib/legco.rb]: Would you mind explaining on the reason why? (Sorry if it's a stupid question)

Comment (siuying) [inline lib/legco.rb]: the original code would assume that there are two phones:

``` ruby
telephone=[nil,nil]
# ...
temp = tr.search("td[3]").text.strip.delete(' ')
telephone[0] = temp.partition("/")[0].strip
telephone[1] = (temp.partition("/")[2].strip==""? nil : temp.partition("/")[2].strip) 
```

while following code assume it is a line of phones, separated by "/"

``` ruby
telephone = []

# ...
telephone = tr.search("td[3]").split("/").collect {|phone| phone.delete(' ') }
```

Before:

``` json
{
    "name": "曾鈺成",
...
    "telephone": [
      "39193001",
      null
    ],
...
  }
```

After:

``` json
{
    "name": "曾鈺成",
...
    "telephone": [
      "39193001"
    ],
...
  }
```

Comment (kenyiu) [inline lib/legco.rb]: Thanks for explanation, I've changed the code.
Feel free to tell me if there is any problems.

Cheers.

**Comment by siuying** [comment]
Thanks for the update. They look awesome!

**Pull request merged**